_sass/_09_elements.scss

```diff
@@ -16,22 +16,6 @@
 /* Custom
 ------------------------------------------------------------------- */
 
-#slack-loading {
-  width: 300px;
-  height: 300px;
-  margin: auto;
-  overflow: hidden;
-  text-align: center;
-  font-weight: bold;
-}
-
-#slack-invite-wrapper {
-  text-align: center;
-}
-#slack-invite {
-  display: none;
-}
-
```

Comment:
I think this has been gone for a long time. 😆 

iirc, this was back when there wasn't Slack invite URLs and I had a "hosted" Slack invite system.